src/components/Timeline/TimelineCards/TimelineCard.tsx

```diff
@@ -29,48 +29,44 @@ const TimelineCard: FC<ITimelineCardProps> = ({ id, cardInfo }) => {
         }
     }
 
+    const timelineStyle = classNames(
+        s.cdTimelineBlock,
+        { [s.reverse]: id % 2 === 0 }
+    )
+
+    const cardContainer = classNames(
```

Comment:
You can add style or className postfixe for the next constants like you did for const timelineStyle to understand it is a style/className